Source/WebKit/GPUProcess/graphics/WebGPU/RemoteBindGroup.cpp

```diff
@@ -46,6 +46,11 @@ RemoteBindGroup::RemoteBindGroup(PAL::WebGPU::BindGroup& bindGroup, WebGPU::Obje
 
 RemoteBindGroup::~RemoteBindGroup() = default;
 
+void RemoteBindGroup::destroy()
```

Comment:
There are already a bunch of destroy functions which are WebGPU API calls; I recommend calling our own internal lifetime management functions something else.

Reply:
how about `destruct`?